content/de/docs/setup/_index.md

```diff
@@ -7,76 +7,96 @@ content_template: templates/concept
-
-Wählen Sie eine [schlüsselfertige On-Premises-Lösung](/de/docs/setup/) aus.
-
-## Individuelle Lösungen
-
-Individuelle Lösungen geben Ihnen die größte Freiheit in Ihren Clustern, erfordern jedoch das meiste Know-how.
-Diese Lösungen reichen von Bare-Metal-Anbietern bis hin zu Cloud-Anbietern mit unterschiedlichen Betriebssystemen.
+{{% /capture %}}
 
-Wählen Sie eine [individuelle Lösung](/de/docs/setup/) aus.
+{{% capture body %}}
 
-{{% /capture %}}
+## Lernumgebung
+
+Benutzen Sie eine Docker-basierende Lösung, wenn Sie Kubernetes erlernen wollen: Von der Kubernetes-Community unterstützte Werkzeuge oder Werkzeuge in einem Ökosystem zum Einrichten eines Kubernetes-Clusters auf einer lokalen Maschine.
+
+{{< table caption="Local machine solutions table that lists the tools supported by the community and the ecosystem to deploy Kubernetes." >}}
+
+|Community           |Ökosystem     |
+| ------------       | --------     |
+| [Minikube](/docs/setup/learning-environment/minikube/) | [CDK on LXD](https://www.ubuntu.com/kubernetes/docs/install-local) |
+| [kind (Kubernetes IN Docker)](https://github.com/kubernetes-sigs/kind) | [Docker Desktop](https://www.docker.com/products/docker-desktop)|
+|                     | [Minishift](https://docs.okd.io/latest/minishift/)|
+|                     | [MicroK8s](https://microk8s.io/)|
+|                     | [IBM Cloud Private-CE (Community Edition)](https://github.com/IBM/deploy-ibm-cloud-private) |
+|                     | [IBM Cloud Private-CE (Community Edition) on Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers)|
+|                     | [k3s](https://k3s.io)|
+
+
+## Produktionsumgebung
+
+Überlegen Sie sich bei der Bewertung einer Lösung für eine Produktionsumgebung, welche Aspekte des Betriebs eines Kubernetes-Clusters (oder von _abstractions_) Sie selbst verwalten oder an einen Anbieter auslagern möchten.
+
+Einige mögliche Abstraktionen von Kubernetes-Clustern sind  {{< glossary_tooltip text="applications" term_id="applications" >}}, {{< glossary_tooltip text="data plane" term_id="data-plane" >}}, {{< glossary_tooltip text="control plane" term_id="control-plane" >}}, {{< glossary_tooltip text="cluster infrastructure" term_id="cluster-infrastructure" >}} und {{< glossary_tooltip text="cluster operations" term_id="cluster-operations" >}}.
+
+Das folgende Diagramm zeigt die möglichen Abstraktionen eines Kubernetes-Clusters und ob eine Abstraktion selbst verwaltet oder von einem Anbieter verwaltet wird.
+
+Lösungen für Produktionsumgebungen![Lösungen für Produktionsumgebungen](/images/docs/KubernetesSolutions.svg)
+
+{{< table caption="Tabelle für Produktionsumgebungs-Lösungen listet Anbieter und deren Lösungen auf." >}}
+
+Die folgende Tabelle für Produktionsumgebungs-Lösungen listet Anbieter und deren Lösungen auf.
+
+|Providers  | Managed | Turnkey cloud  | On-Prem Datacenter  | Custom (cloud) | Custom (On-premises VMs)| Custom (Bare Metal) |
```

Comment:
These headings are not translated. Is that intentional?

Reply:
I left it as it is (tooltip display text and table headings). If you look at the test output [1] this text corresponds with the content in the image. We can build our own SVG in German but I think thats not easy to maintain in the future.

[1] https://deploy-preview-17966--kubernetes-io-master-staging.netlify.com/de/docs/setup/

Reply:
Ah OK. It would be better to localize that image; ideally, there would be tooling to make that easy (right now, there is not).

Reply:
I investigated it a little bit. It's possible to extend the SVG in multi-language style

```
<switch>
    <text
       systemLanguage="de"
       german text
     </text>
    <text
       systemLanguage="en"
       english text
     </text>
</switch>
```

But that would be a very big file in multi-lang and length and size must be adjusted for each text string. Another way would be to use a markdown table instead SVG. Otherwise in Japanese and Korean are already used the origin, maybe it's also a way for German :)

Reply:
Hi @sftim,
I want to circle back to this PR. After one month it seems I had no time to work on this SVG stuff. The technical solution should be clear, but I'm not sure if this is common between all language teams. My little hope would be to merge this PR at it is (it's also my first one) and work on the SVG topic later. What do you think?

Reply:
That's fine. I hope the German-language localization team agree.
It is usually quite trick to localize images; it's an ideal and in my view it _is_ fine to merge PRs that leave images not yet localized.

Reply:
I am ok with keeping the SVG in English. @rlenferink @remyleone what do you think?

Reply:
I'd say keep the SVG in English because (as Remy said) it's indeed easier to maintain.